README.md

```diff
@@ -101,6 +101,20 @@ that iterates through them:
 order.calculate_total_price(double(:price => 1.99), double(:price => 2.99))
 ```
 
+### Stubbing a chain of methods
+
+You can use `receive_message_chain` in place of `receive` to stub a chain of messages:
+
+```ruby
+allow(double).to receive_message_chain(:foo, :bar) { :baz }
```

Comment:
I think it's fine that you showed the 3 ways that work before.  Any reason you removed it?  I didn't mean to suggest you remove them; in my suggested snippet, the `# ...` was a standin because I was too lazy :).